CHANGELOG.md

```diff
@@ -33,12 +33,16 @@ read the notes of the `Enso 2.0.0-alpha.1` release.
 - [Added the ability to reposition visualisations.][1096] There is now an icon in the visualization 
   action bar that allows dragging the visualization. Once the visualization has been moved, there 
   appears another icon that will reset the position to the original position.
+- [Allow Tables to feed the Geo Map visualisation.][1187] Tables that have `latitude`, `longitude`
+  and `label` columns can now be shown in a Geo Map visualisation where each row is mapped to a 
```

Comment:
why this restriction? I mean, why tables without label could not be shown on a map?

Reply:
This was not documented correctly: labels are optional.